geps/gep-1897/index.md

```diff
@@ -5,61 +5,51 @@
 
 ## TLDR
 
-This document specifically addresses the topic of conveying HTTPS from the Gateway
-dataplane to the backend (backend TLS termination), and intends to satisfy the single
-use case “As a client implementation of Gateway API, I need to know how to connect to
-a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
-so in order to drive resolution this GEP focuses only on this single piece of functionality.
+This document aims to address the use case of origination of TLS to backends from implementations.
+This is accomplished by a new `BackendTLSPolicy` API.
 
-Furthermore, for Gateway API to handle the case where the service or backend owner is doing their own TLS, _and_
-the service or backend owner wants to validate the clients connecting to it, two things need to happen:
+This API solves the question "As an implementation, when I establish a connection to a backend, should I originate a TLS connection? If so, with what properties?"
 
-- The service or backend owner has to provide a method for the Gateway owner to retrieve a certificate.
-- Gateway API has to provide a way for the Gateway to configure and apply the validation options.
+## Goals
 
-## Immediate Goals
+This API aims to solve use cases around connecting to services that serve HTTPS/TLS traffic.
+These could be in-cluster applications (typically a `Service`) or external services (such as `https://google.com`).
+In many cases, communication going through gateway implementations is not TLS encrypted, so we expect the implementation itself to *originate* the TLS directly.
+This is in contrast with the TLS Passthrough cases, which remain supported.
+
+Some common examples that would utilize this API:
+* A Gateway terminates HTTPS, does routing based on the HTTP traffic, then originates HTTPS to the backend.
+* A service mesh accepts plaintext traffic (`http://google.com:80`) from a client, then originates HTTPS to that destination (`https://google.com:443`).
```

Comment:
Hang on a sec, is this for _outbound_ client requests? That are leaving the cluster? Is this Egress config by default? What does the backend Service look like in this case?

I can totally see how this config _could_ be used for that, but including `ExternalName` Services is a _very_ large change here, and I'm not sure we should open that can of worms yet (if that's what you mean).

Reply:
+1 to everything john said. FWIW - We (GKE) intend to do something similar for external services.

Reply:
It's not a goal or an API proposal - it's just a statement of how many of the implementations *will* and *already do* use the API. I can remove it from the GEP if we do not want that information I suppose

Reply:
That's great, but this GEP is already increasing quite a bit in scope, I'd rather keep it tighter please.